core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss

```diff
@@ -24,10 +24,8 @@
     height: 100%;
 }
 
-.content-search__show-query-dialog {
-    .dijitDialogTitleBar {
-        z-index: 1;
-    }
+#queryResults {
+    padding: 16px;
 }
```

Comment:
Why do we need this?

Reply:
I need space under the query box on content search dialog